### DISCUSSION TITLE
Would it be possible to render ZN1/ZN2 games in MAME?

### DISCUSSION THREAD

**Discussion by youtubeeicarlos** [mamedev/discussions#68] [Ideas]
If the ZN1/ZN2 games use the base PS1 motherboard, why aren't they rendered by a DuckStation or ePSXe graphics plugin in MAME? MAME for this type of game does not offer the best graphics options (and I tested them all, and they have regrettable aliasing in 3D games) I also have a "little theory", that it would be possible to emulate Gundam Seed: Federation vs.  Z.A.F.T.  and the Mobile Suit Z-Gundam: A.E.U.G.  vs Titans DX from Arcade, using the PS2 motherboard as a base, since the PS1 motherboard is used in the ZN1/ZN2, with this type of "partnership" that Capcom does with Sony, I don't think it's impossible that These two games become playable, since they both have a PS2 version and their arcade version is almost the same size as this one (with the only difference being that the Arcade version is an update with more content than there is in the version of PS2).

**Comment by rb6502** [comment]
MAME displays the games as they originally appeared.  If you can't handle them as they originally appeared (and I'll freely admit that 2D games have aged a lot more gracefully than early 3D) don't play them in MAME.

**Reply by youtubeeicarlos** [reply]
It's not like it's possible to play anywhere else... I was referring to some way of rendering with better quality, maybe at 2X, 4X, using more of the video card's potential
